test/PriceConsumerV3_test.js

```diff
@@ -1,23 +1,29 @@
 const { expect } = require('chai')
+const chai = require('chai')
+const BN = require('bn.js');
+chai.use(require('chai-bn')(BN));
 const { deployments, getChainId } = require('hardhat')
 const { networkConfig } = require('../helper-hardhat-config')
 
 describe('PriceConsumer', async function () {
   // Price Feed Address, values can be obtained at https://docs.chain.link/docs/reference-contracts
-  let priceConsumerV3, ethUsdAggregator
+  let priceConsumerV3, ethUsdAggregator, PriceConsumerV3, MockV3Aggregator
 
   beforeEach(async () => {
-    await deployments.fixture(['mocks', 'feed'])
+    // We ONLY deploy the fixtures on a local test
+    if (await getChainId() == 31337) {
+      await deployments.fixture(['mocks', 'feed'])
+    }
```

Comment:
You should only be running unit tests on local networks.

Reply:
Should we have a separate section for running integration tests? I 100% will want to test on a testnet after running my tests locally before.